src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -382,6 +386,11 @@ interfaces:
     field_name_patterns:
       resource: book
     timeout_millis: 60000
+    surface_treatments:
+    - include_languages:
+      - go
+      treatments:
+      - visibility: DISABLED
```

Comment:
what is the motivation for using oneof? It seems like it could be slightly simpler this way:

```
  surface_treatments:
  - include_languages:
    - go
    visibility: DISABLED
```